src/uid2Sdk.ts

```diff
@@ -219,11 +221,17 @@ export class UID2 {
 
     this._opts = opts;
     this._cookieManager = new UID2CookieManager({ ...opts });
+    this._localStorageManager = new UID2LocalStorageManager();
     this._apiClient = new Uid2ApiClient(opts);
     this._tokenPromiseHandler.registerApiClient(this._apiClient);
-    const identity = this._opts.identity
-      ? this._opts.identity
-      : this._cookieManager.loadIdentityFromCookie();
+
+    let identity;
+    if (this._opts.identity) {
+      identity = this._opts.identity;
+    } else {
+      const localStorageIdentity = this._localStorageManager.loadIdentityFromLocalStorage();
+      identity = localStorageIdentity !== null ? localStorageIdentity : this._cookieManager.loadIdentityFromCookie();
```

Comment:
You could probably use ?? to simplify this a bit.

Reply:
I've changed how this works